dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -188,3 +188,6 @@ excludeAllHandledTypesClasses.desc=When this property is set to true, during sta
 
 skipEncodedCharVerification.name=Skip verification of encoded characters in URI
 skipEncodedCharVerification.desc=A request is rejected if the request URI contains the %23, %2E, %2F, or %5C encoded characters. When this property is set to true, the request URI is not checked for these encoded characters. This property is available starting in servlet 6.0, and default value is false.
+
+maxFileCount.name=The maximum uploaded files in a single request
+maxFileCount.desc=The maximum number of files that can be uploaded by a multipart/form-data request. The default is 5000; setting to -1 for unlimited files upload.
```

Comment:
ID suggested rewrite:
maxFileCount.desc=The maximum number of files that can be uploaded by a multipart/form-data request. The default value is 5000. For unlimited file upload, set the value to -1.